sormas-backend/src/test/resources/testcontainers/Dockerfile

```diff
@@ -1,9 +1,10 @@
 FROM postgres:10.18-alpine
 
-RUN apk add --no-cache --repository http://dl-cdn.alpinelinux.org/alpine/v3.13/main/ musl=1.2.2-r1 musl-dev=1.2.2-r1 && \
-    apk add --no-cache openssl=1.1.1o-r0 curl=7.79.1-r1 tzdata=2022a-r0 py-pip=20.3.4-r1 python3-dev=3.9.5-r2  \
-    postgresql-dev=13.6-r1 postgresql-contrib=13.6-r1 make=4.3-r0 gcc=10.3.1_git20210424-r2 py3-psutil=5.8.0-r1  && \
-    pip install --no-cache-dir pgxnclient==1.3.2 && \
+# hadolint ignore=DL3018,DL3013
+RUN apk add --no-cache --repository http://dl-cdn.alpinelinux.org/alpine/v3.13/main/ musl musl-dev && \
```

Comment:
We could define the `musl` dependencies as they are in the [SORMAS-Docker](https://github.com/hzi-braunschweig/SORMAS-Docker/blob/development/postgres/Dockerfile) and then we will have a similar setup with it and from my point of view it can be merged then